-------------------------------------------------------------------------------
Test set: com.example.MainTest
-------------------------------------------------------------------------------
Tests run: 6, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.089 s <<< FAILURE! -- in com.example.MainTest
com.example.MainTest.VI_should_return_6 -- Time elapsed: 0.008 s <<< FAILURE!
org.opentest4j.AssertionFailedError: 

expected: 6
 but was: 2
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502)
	at com.example.MainTest.VI_should_return_6(MainTest.java:41)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)

